README.md

```diff
@@ -49,7 +51,7 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:castle][:guests].push town[:residents].delete 'Belle'
 ```
```

Comment:
This is great, functional, but i think parentheses will really help with syntactic clarity of code.

`town[:castle][:guests].push(town[:residents].delete('Belle'))